buidl/schnorr.py

```diff
@@ -1,16 +0,0 @@
-from buidl.helper import sha256
-
-
-TAGS = {
-    "aux": sha256(b"BIP0340/aux") + sha256(b"BIP0340/aux"),
-    "nonce": sha256(b"BIP0340/nonce") + sha256(b"BIP0340/nonce"),
-    "challenge": sha256(b"BIP0340/challenge") + sha256(b"BIP0340/challenge"),
-}
-
-
-def tagged_hash(tag, msg):
-    return sha256(TAGS[tag] + msg)
-
-
-def xor_bytes(a, b):
-    return bytes(x ^ y for x, y in zip(a, b))
```

Comment:
Why did you move this to `helper.py`? It seems better as you had it before in its own module.

Reply:
I realized that the name of the module was wrong. It's not just schnorr, but all the hash functions are now all over the place as part of Taproot. It would be silly to have a file for the single xor function.

Reply:
That's how I had it originally. But then that created a lot more circular imports.